frontend/fingertips-frontend/mock/data/healthdata.ts

```diff
@@ -1950,6 +1954,7 @@ export const mockHealthData: Record<string, HealthDataForArea[]> = {
           ageBand: '0-4',
           sex: 'Persons',
           trend: HealthDataPointTrendEnum.NotYetCalculated,
+          isAggregate: true,
```

Comment:
I don't think this one would have `isAggregate` set to true (as it has an age value, and the age values aren't all the same for all data points).
```suggestion
          isAggregate: false,
```